jwst/cube_build/ifu_cube.py

```diff
@@ -1408,35 +1662,41 @@ def setup_ifucube_wcs(self):
         final_b_min = min(corner_b)
         final_b_max = max(corner_b)
 
-        log.debug(f'final a and b:{final_a_min, final_b_min, final_a_max, final_b_max}')
-        log.debug(f' min and max wavelengths:   {min(lambda_min), max(lambda_max)}')
+        log.debug(f"final a and b:{final_a_min, final_b_min, final_a_max, final_b_max}")
+        log.debug(f" min and max wavelengths:   {min(lambda_min), max(lambda_max)}")
         # ______________________________________________________________________
         # the wavelength limits of cube are determined from 1. User or 2. cubepars
         # reference file (in the priority)
         final_lambda_min = self.wavemin
         final_lambda_max = self.wavemax
 
-        log.debug(f' final min and max used in IFUcube:   {final_lambda_min, final_lambda_max}')
+        log.debug(f" final min and max used in IFUcube: {final_lambda_min, final_lambda_max}")
 
-        if self.instrument == 'MIRI' and self.coord_system == 'internal_cal':
+        if self.instrument == "MIRI" and self.coord_system == "internal_cal":
             #  we have a 1 to 1 mapping in y across slice  dimension
-            nslice = self.instrument_info.GetNSlice(parameter1[0])
-            log.info(f'Across slice scale {self.cdelt2}')
+            nslice = self.instrument_info.get_nslice(parameter1[0])
+            log.info(f"Across slice scale {self.cdelt2}")
             self.cdelt2 = (final_b_max - final_b_min) / nslice
 
             final_b_max = final_b_min + (nslice) * self.cdelt2
-            log.info('Changed the across slice scale dimension so we have 1-1 mapping between b and slice #')
-            log.info(f'New across slice scale {self.cdelt2}')
+            log.info(
+                "Changed the across slice scale dimension so we have 1-1 "
+                " mapping between b and slice #"
+            )
+            log.info(f"New across slice scale {self.cdelt2}")
 
         # ______________________________________________________________________
-        if self.instrument == 'NIRSPEC' and self.coord_system == 'internal_cal':
+        if self.instrument == "NIRSPEC" and self.coord_system == "internal_cal":
             #  we have a 1 to 1 mapping in x - across slice  dimension.
             nslice = 30
-            log.info(f'Across slice scale {self.cdelt1}')
+            log.info(f"Across slice scale {self.cdelt1}")
             self.cdelt1 = (final_b_max - final_b_min) / nslice
             final_b_max = final_b_min + (nslice) * self.cdelt1
-            log.info('Changed the across slice scale dimension so we have 1-1 mapping between b and slice #')
-            log.info(f'New across slice Scale {self.cdelt1}')
+            log.info(
+                "Changed the across slice scale dimension so we have 1-1 "
```

Comment:
There doesn't need to be a space at the end of the first string and the beginning of the second. That puts two spaces in the string, rather than a single space.